pkg/tfbridge/diff_test.go

```diff
@@ -1539,6 +1544,8 @@ func TestNestedComputedSetIntUpdateReplace(t *testing.T) {
 }
 
 func TestNestedComputedSetIntAdd(t *testing.T) {
+	// TODO[pulumi/pulumi-terraform-bridge#2428]: Incorrect detailed diff with unknown elements
+	t.Skip("Skipping until pulumi/pulumi-terraform-bridge#2428 is resolved")
```

Comment:
2428 is https://github.com/pulumi/pulumi-terraform-bridge/issues/2428 this is the wrong number perhaps?